### DISCUSSION TITLE
Links are not clickable

### DISCUSSION THREAD

**Discussion by vkolotov** [afconsult/quill-mention#304] [Q&A] [answered]
When a mention object has a link attribute, the module inserts <a> tag, however the tag is not clickable (nothing happens when you click on mention element in quill).

I can see that there is an even listener is added to <span> element:

![image](https://user-images.githubusercontent.com/1161883/125650679-c3d9d77f-9912-4528-8962-abb738bbebcf.png)

This even listener stops normal event from happening, ie. links are not clickable.

Is there any reason why this listener is added? I can't spot anywhere in the code where it is used.

**Comment by jeffdrumgod** [comment, marked as answer]
Temporary adjustment until package validation: Add this code snippet to your project. (Adjust as per your need)

```typescript
window.addEventListener('mention-clicked', function ({ value }: any) {
  if (value?.link) {
    window.open(value.link, value?.target ?? '_self');
  }
});
```
